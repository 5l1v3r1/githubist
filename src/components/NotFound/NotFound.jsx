// @flow

import React from 'react';

import { LinkButton, ServerStatus } from '..';

import styles from './NotFound.scss';

const NotFound = () => (
  <ServerStatus code={404}>
    <div className={styles.notFound}>
      <p className={styles.text}>Aradığınız sayfa bulunamadı.</p>
      <LinkButton to="/" primary>
        Anasayfaya Dön
      </LinkButton>
    </div>
  </ServerStatus>
);

export default NotFound;
