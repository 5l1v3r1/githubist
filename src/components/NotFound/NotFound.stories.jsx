// @flow

import React from 'react';
import { storiesOf } from '@storybook/react';
import { MemoryRouter } from 'react-router-dom';
import backgrounds from '@storybook/addon-backgrounds';

import NotFound from './NotFound';

storiesOf('NotFound', module)
  .addDecorator(backgrounds([{ name: 'dark', value: '#2d2e33', default: true }]))
  .add('default', () => (
    <div style={{ backgroundColor: '#fff' }}>
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>
    </div>
  ));
