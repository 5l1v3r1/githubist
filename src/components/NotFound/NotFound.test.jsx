// @flow

import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './NotFound';

it('renders correctly', () => {
  const wrapper = shallow(<NotFound />);

  expect(wrapper).toMatchSnapshot();
});
