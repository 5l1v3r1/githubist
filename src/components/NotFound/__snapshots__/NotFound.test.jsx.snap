// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Status
  code={404}
>
  <div>
    <p>
      Aradığınız sayfa bulunamadı.
    </p>
    <LinkButton
      className=""
      primary={true}
      secondary={false}
      small={false}
      to="/"
    >
      Anasayfaya Dön
    </LinkButton>
  </div>
</Status>
`;
